irohad/model/impl/query_execution.cpp

```diff
@@ -283,8 +317,9 @@ iroha::model::QueryProcessingFactory::execute(
     return executeGetAccountTransactions(*qry);
   }
   if (instanceof <iroha::model::GetAccountAssetTransactions>(query.get())) {
-    auto qry = std::static_pointer_cast<
-        const iroha::model::GetAccountAssetTransactions>(query);
+    auto qry = std::
+        static_pointer_cast<const iroha::model::GetAccountAssetTransactions>(
```

Comment:
There's:
https://github.com/hyperledger/iroha/blob/b7f988b7a21fa23e6e68be6d992eaf8a3944d957/irohad/model/impl/query_execution.cpp#L33
So no need to write full namespace